src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -260,6 +260,7 @@ const SearchFieldList = ({ onSearch }: Props) => {
       setSearchParams((sp) => ({
         ...sp,
         class_standing: '',
+        minor: '',
```

Comment:
Alumni's Minors are not saved, so the results would always be empty if graduation year was searched together with a minor